src/helpers/LaunchCamera.tsx

```diff
@@ -42,7 +43,13 @@ const takePicture = async (
           setResponse,
         );
       } else {
-        Alert.alert('Camera permission denied');
+        Toast.show({
+          type: 'error',
+          text1: 'Permission denied',
+          text2: 'Camera Permission denied',
```

Comment:
Move 'Camera Permission denied' to en.ts file and then use the string over here

Reply:
I will do it